mymod.py

```diff
@@ -0,0 +1,13 @@
+def count_lines(name):
+    func = open(name,'r')
```

Comment:
It's better to use `with` operator for opening file, because using it you don't bother about to closing a file.
As you see, you've missed this.